go/stellar/note_test.go

```diff
@@ -65,7 +64,9 @@ func TestNoteLengthLimit(t *testing.T) {
 
 	// verify we can encrypt content with max length
 	pre := sampleNote()
-	pre.Note = strings.Repeat(".", libkb.MaxStellarPaymentNoteLength)
+	note, err := libkb.RandString("", libkb.MaxStellarPaymentNoteLength)
```

Comment:
ah, was the issue that '...' repeated had an encoded length shorter than others?

Reply:
yeah :/